modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -153,6 +220,9 @@ def run
       end
     end
 
+    puts @valid_modules_table if datastore['PrintTables']
+    puts @invalid_modules_table if datastore['PrintTables']
+
```

Comment:
This should be a `print_*` instead of `puts` as warned by `msftidy`, consider using `print_status` or even consider removing the `PrintTables` option and just making this `vprint_status`?

```suggestion
    vprint_status @valid_modules_table
    vprint_status @invalid_modules_table

```